miniconf/src/packed.rs

```diff
@@ -151,26 +162,36 @@ impl Packed {
     /// Return the representation aligned to the LSB with the marker bit
     /// moved from the LSB to the MSB.
     #[inline]
-    pub fn into_lsb(self) -> NonZeroUsize {
-        // Note(unwrap): we ensure there is at least the marker bit set
-        NonZeroUsize::new(((self.get() >> 1) | (1 << Self::CAPACITY)) >> self.trailing_zeros())
-            .unwrap()
+    pub const fn into_lsb(self) -> NonZeroUsize {
+        match NonZeroUsize::new(
+            ((self.0.get() >> 1) | (1 << Self::CAPACITY)) >> self.0.trailing_zeros(),
+        ) {
+            Some(v) => v,
+            // We ensure there is at least the marker bit set
+            None => unreachable!(),
```

Comment:
Why not unwrap instead? Not sure why this is being changed

Reply:
not const

Reply:
Ah okay, thanks!